docs/index.rst

```diff
@@ -1,17 +1,67 @@
 .. include:: ../README.rst
    :end-before: About
 
+
 .. include:: ../README.rst
-   :start-after: -----
+   :start-line: 25
    :end-before: Features
 
-.. toctree::
-   :numbered:
 
-   introduction
-   cli
-   restapi
-   contributing
-   changes
-   license
-   authors
+.. include:: ../README.rst
+   :start-line: 32
+   :end-before: Useful links
+
+**Contents**
+
+.. contents::
+   :local:
+   :depth: 1
+   :backlinks: none
+
+
+CLI API
+=======
+
+.. click:: reana_server.reana_admin:reana_admin
+   :prog: flask reana-admin
+   :show-nested:
+
+
+REST API
+========
+
+The REANA Server offers a REST API for management workloads
+(workflows, jobs, tasks, etc.) running on REANA Cloud.
+Detailed REST API documentation can be found `here <_static/api.html>`_.
+
+.. automodule:: reana_server.rest.ping
+   :members:
+
+.. automodule:: reana_server.rest.users
+   :members:
+
+.. automodule:: reana_server.rest.workflows
+   :members:
+
+
+.. include:: ../CONTRIBUTING.rst
```

Comment:
We can simplify Contributing by killing difference between Issues and Pull requests in order to have one simple wall of text.  We can do it by editing CONTRIBUTING.rst and killing sections there. We could reword beginning of paragraphs to be:

- Bug reports, issues, feature requests, and other contributions are...

- If you create a pull request fixing a bug or implementing a feature, you can...